pkg/apis/deployment/v1alpha/register.go

```diff
@@ -31,7 +31,11 @@ import (
 const (
 	ArangoDeploymentResourceKind   = "ArangoDeployment"
 	ArangoDeploymentResourcePlural = "arangodeployments"
-	groupName                      = "database.arangodb.com"
+
+	ArangoBackupResourceKind   = "ArangoBackup"
+	ArangoBackupResourcePlural = "arangobackups"
+
+	groupName = "database.arangodb.com"
```

Comment:
Maybe we should move the new code to `pkg/apis/backup/v1alpha1` to get correct semantic versioning.

Reply:
We cannot. Problem is that "deployment" directory is registering himself as "backup" in ClientSets. Changing this would create backward compatibility issues, which are not expected.